src/ChunkSender.h

```diff
@@ -53,87 +56,73 @@ class cChunkSender:
 {
 	typedef cIsThread super;
 public:
-	cChunkSender(void);
+	cChunkSender(cWorld & a_World);
 	~cChunkSender();
 
 	enum eChunkPriority
 	{
-		E_CHUNK_PRIORITY_HIGH   = 0,
-		E_CHUNK_PRIORITY_MEDIUM = 1,
-		E_CHUNK_PRIORITY_LOW    = 2,
+		E_CHUNK_PRIORITY_LEGENDARY = 0,
```

Comment:
About this, it was added to push the original HIGH higher, to fit a priority between high and medium - this was the position where the old m_ChunksReady list was processed, which was where cNotifyChunkSender put its chunks. Is it possible to just have three priorities, and give cNotifyChunkSender a MEDIUM priority?